src/parse-transaction-log.ts

```diff
@@ -70,6 +70,11 @@ export function parseTransactionLog(path: string): TransactionLog {
 
 		while (fileOffset < size) {
 			const timestamp = read(8).readDoubleBE(0);
+			if (timestamp === 0) {
+				// if we encounter zero padding, we can stop reading the entries since the next entry will start at the next 8-byte boundary, which is the same as the current file offset.
+				size = fileOffset - 8;
+				break;
+			}
```

Comment:
What's the point of this? `timestamp` should never be zero. Commenting out this change has no effect on the tests.

Reply:
I did comment it out and tested it on macOS and the tests worked. I'll test removing it on Windows tomorrow after this branch is stable again.